java/ql/src/Security/CWE/CWE-489/WebviewDebuggingEnabled.ql

```diff
@@ -0,0 +1,19 @@
+/**
+ * @name Android Webview debugging enabled
+ * @description Webview debugging should not be enabled in production builds.
```

Comment:
```suggestion
 * @description Enabling Webview debugging in production builds can expose entry points or leak sensitive information.
```

Just a small change to bring in line with the [query style guide](https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#query-descriptions-description).